### ISSUE TITLE
[v7] White screen on login to backend if not visiting front end first

### ISSUE THREAD

**Issue by arknu** [umbraco/Umbraco-CMS#9696]
I just again hit the issue described in this forum thread: https://our.umbraco.com/forum/using-umbraco-and-getting-started/85747-417-missing-token-suddenly-occurring-on-all-our-sites

As mentioned in this post (https://our.umbraco.com/forum/using-umbraco-and-getting-started/85747-417-missing-token-suddenly-occurring-on-all-our-sites#comment-307140) this seems to be due to various session cookies not being set if you go directly to the backend without first hitting the frontend. If you hit the frontend and *then* go to the backend, things start working.

So, it seems something is assuming a session cookie will be set when it isn't.

I'm hoping to do a deeper analysis and hopefully a PR, but I won't have time to do that for a few weeks.

## Umbraco version
7.15.6

v8 is not affected

## Steps to reproduce
1. Clear all cookies for the site in question
2. Go directly to Umbraco (i.e. hit site.com/umbraco)
3. Log in 

### Expected result
The backend should load normally

### Actual result
The backend doesn't load, only a white screen is displayed
A bunch of HTTP417 Missing token errors appear in the console. These happen because the various validation cookies never get set.

**Comment by nul800sebastiaan** [comment]
Thanks @arknu, I don't know how to reproduce this. I have opened a v7 site (frontend), cleared all cookies, then went to the `/umbraco` route and the login screen works just fine. So there must be more to it than just a missing cookie. I believe the backoffice is already responsible for handing out relevant cookies anyway, and I can't see any cookies being added on this site except for a cloudflar one when I clear everything. 

![image](https://user-images.githubusercontent.com/304656/105829092-360d4480-5fc4-11eb-9aad-b82f1a696a24.png)

I'm happy for you to try to fix it, but I doubt that a frontend cookie is the cause. 
Just note: there's no need to report an issue to send a PR, we are happy to look at PRs that have no related issue.

With that said, I will close this as a duplicate of https://github.com/umbraco/Umbraco-CMS/issues/4138. 
As said, really happy for you to have a look and figure out a fix, but make sure that first we can replicate the issue otherwise it's going to be difficult to evaluate a fix. 🙈

**Comment by nul800sebastiaan** [comment]
Additional note: the login screen doesn't require any additional cookies either, again, I only see the cloudflare cookie when on the login screen. Only once the login succeeds do we get specific `UMB-` cookies added.
